app/models/customer.py

```diff
@@ -0,0 +1,24 @@
+from sqlalchemy.orm import relationship
```

Comment:
We can actually get rid of this! We have access to `relationship` module through `db` just like we do with with `db.Column` or `db.String`

```suggestion

```